bindata/assets/alerts/cpu-utilization.yaml

```diff
@@ -69,3 +69,13 @@ spec:
           labels:
             namespace: openshift-kube-apiserver
             severity: critical
+        - alert: highOverallControlPlaneIOWait
+          annotations:
+            description: Control-plane nodes are spending more than 20% of CPU time on iowait mode.
+            summary: CPU utilization on master nodes is more than 20% on iowait mode.
+          expr: |
+            (sum(irate(node_cpu_seconds_total {mode="iowait"} [2m])) without (cpu)) / count(node_cpu_seconds_total) without (cpu) * 100 > 20
```

Comment:
Hi @openshift/sre-alert-sme, we are trying to build an alert to detect disk/network pressure causing CPU latency. Based on your experience with OCP, would 20% of CPU time waiting for IO make sense to start alerting on?

Also what could be a meaningful threshold for a critical alert?